dbos/_sys_db.py

```diff
@@ -577,51 +580,31 @@ def get_workflow_status(
             }
             return status
 
-    def await_workflow_result_internal(self, workflow_uuid: str) -> dict[str, Any]:
-        polling_interval_secs: float = 1.000
-
+    def await_workflow_result(self, workflow_id: str) -> Any:
         while True:
             with self.engine.begin() as c:
```

Comment:
Does this even need to be an transaction?

Reply:
It's a single statement, there's no transaction over the loop

Reply:
Isn't every iteration 3 requests? Begin / select / commit?